library/Icinga/Application/Config.php

```diff
@@ -412,17 +412,16 @@ public static function app($configname = 'config', $fromDisk = false)
      */
     public static function module($modulename, $configname = 'config', $fromDisk = false)
     {
-        if (! isset(self::$modules[$modulename])) {
-            self::$modules[$modulename] = array();
+        if (! isset(static::$modules[$modulename])) {
+            static::$modules[$modulename] = array();
         }
 
-        $moduleConfigs = self::$modules[$modulename];
```

Comment:
This line contains the original bug, an array is copied as such and not as a reference